ci/Dockerfile.multi

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+###
+# JNI CI image for multi-platform build
+#
+# Arguments: CUDA_VERSION=11.8.0
+#
+###
+ARG CUDA_VERSION=11.8.0
+ARG OS_RELEASE=8
+# multi-platform build with: docker buildx build --platform linux/arm64,linux/amd64 <ARGS> on either amd64 or arm64 host
+# check available offcial arm-based docker images at https://hub.docker.com/r/nvidia/cuda/tags (OS/ARCH)
+FROM --platform=$TARGETPLATFORM nvidia/cuda:$CUDA_VERSION-devel-rockylinux$OS_RELEASE
```

Comment:
I believe the GH systems will come with Ubuntu, can we build and test with Ubuntu 22.04?

Reply:
This would be more effort as we need to provide 2 toolsets to support developing on 2 kinds of OS distros (not only RHEL), or we are fine to just test the artifacts on ubuntu?

Both should be fine as we still have enough time to discuss and implement.
Currently I would like to know if the current classifier solution looks good, so I can continue the work, or I may require some help from developers before CICD updates, thanks ~

Reply:
Eventually we need to test on both Rocky9 and Ubuntu 22.04.  My request is to start with Ubuntu 22.04 if we can.

Reply:
> Eventually we need to test on both Rocky9 and Ubuntu 22.04. 

We did test on both Rocky9 and Ubuntu 22.04 for x86, you mean for arm we also need to cover all x86 scenarios?
sounds like we need fill much much more arm instances in our pool.

> My request is to start with Ubuntu 22.04 if we can.

Sure, I will prepare ubuntu ENV for arm build later

Reply:
>> can we build and test with Ubuntu 22.04?

> Sure, I will prepare ubuntu ENV for arm build later

To ensure we're on the same page, the goal should not be to build on all supported OS's but instead build on _one_ OS and run that binary on all supported OS's.  That's why building on rocky makes sense to me, as that's closest to the build setup we had for x86_64 where we build once and run that everywhere.

Reply:
Sorry, my misunderstanding.  Yes we can build on Rocky8 if that is the lowest common denominator, and then test on Ubuntu 22.04.

Reply:
in default cuda11.8-ubuntu22 and cuda11.8-rocky8 arm64 images,

ubuntu22,
```
ldd --version
ldd (Ubuntu GLIBC 2.35-0ubuntu3.1) 2.35
```

rocky8,
```
ldd --version
ldd (GNU libc) 2.28
```

Lets keep rocky8 for building, and cover the tests in ubuntu22 ENV later